src/client/game.cpp

```diff
@@ -1394,16 +1394,14 @@ bool Game::createSingleplayerServer(const std::string &map_dir,
 	std::string bind_str = g_settings->get("bind_address");
 	Address bind_addr(0, 0, 0, 0, port);
 
-	if (g_settings->getBool("ipv6_server")) {
-		bind_addr.setAddress((IPv6AddressBytes *) NULL);
-	}
-
+	if (g_settings->getBool("ipv6_server"))
+		bind_addr.setAddress(static_cast<IPv6AddressBytes *>(NULL));
```

Comment:
for consistency between the two almost identical code sections

```suggestion
		bind_addr.setAddress(static_cast<IPv6AddressBytes *>(nullptr));
```